diff --git a/appcreate/partitionedfs.py b/appcreate/partitionedfs.py
index 50260a2..d9a177d 100644
--- a/appcreate/partitionedfs.py
+++ b/appcreate/partitionedfs.py
@@ -86,7 +86,7 @@ class PartitionedMount(Mount):
                 p['type'] = 'primary'
                 p['num'] = d['numpart']
             if d['offset'] == 0:
-                d['offset'] = 1
+                d['offset'] = 2
             p['start'] = d['offset']
             d['offset'] += p['size']
             d['partitions'].append(n)
